lib/std/heap.zig

```diff
@@ -578,6 +578,14 @@ pub fn StackFallbackAllocator(comptime size: usize) type {
     };
 }
 
+/// Runtime known minimum page size.
+pub fn pageSize() usize {
+    return switch (builtin.os.tag) {
+        .linux => if (builtin.link_libc) @intCast(std.c.sysconf(std.os.linux.SC.PAGESIZE)) else std.os.linux.getauxval(std.elf.AT_PAGESZ),
```

Comment:
This is being called in many hot paths. It should do a thread-safe global memoization of this value, with the hot path doing a single branch on an atomic load of the value.

Reply:
Alright, I added that. Should I use `std.atomic.Atomic`?